src/scripts/analyze_missingness.py

```diff
@@ -92,16 +123,31 @@ def count_records_missing_data(
     covid_loincs = pd.read_csv(
         pathlib.Path(__file__).parent / "assets" / "covid_loinc_reference.csv"
     )
+    # Use known COVID vaccine related codes as described in
+    # https://build.fhir.org/ig/HL7/fhir-COVID19Library-ig/branches/master/ValueSet-covid19-cvx-codes-value-set.html
+    # to filter out any non-covid vaccines
+    covid_vax_codes = pd.read_csv(
+        pathlib.Path(__file__).parent / "assets" / "covid_vax_codes.csv", dtype="object"
+    )
 
-    pre_linkage_covid = pre_linkage.merge(covid_loincs, on="loincCode", how="inner")
-
-    pre_linkage_covid.drop(
-        columns=[
-            c
-            for c in pre_linkage_covid.columns
-            if c not in list(equity_fields.keys()) + ["patientHash"]
-        ],
-        inplace=True,
+    pre_linkage_elr = pre_linkage_elr.merge(covid_loincs, on="loincCode", how="inner")
+    pre_linkage_ecr = pre_linkage_ecr.merge(covid_loincs, on="loincCode", how="inner")
```

Comment:
My understanding was that for ECR we were filtering using both LOINC code and vaccine code, but this implies that we're only filtering using the LOINC code. Is that right?

Reply:
Yes, that's correct. I misspoke earlier when I said we double merged, my bad. In what I've seen the LOINC is a more granular merging criterion to filter out unrelated data, so we thought this was good for our purposes.